cm2350/peripherals/flash.py

```diff
@@ -980,15 +986,18 @@ def write(self, block, offset, bytez):
         # silently do nothing?
 
 
-class FLASH(mmio.MMIO_DEVICE):
+class FLASH(Peripheral, MMIO_DEVICE):
```

Comment:
ummm...  So, does this all need to change to Module?  (there are 50 mentions of Peripheral in this PR)

Reply:
oh! ... I thought I changed it, lemme check

Reply:
oh whoops, yes eventually but not until I rebase this PR off of main.